### PR TITLE
added openapi spec for all getters

### PR DESCRIPTION
**Pull request by NewtTheWolf**

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                type: array
+                items:
+                  $ref: "#/components/schemas/Note"
+          description: An array of notes
+      summary: Get list of notes for the team
+components:
+  schemas:
+    Team:
+      type: object
+      properties:
```
Comment (joerichsen) [inline openapi.yml:143]: The properties should correspond to the following data structure

```
    {
        "id": 6237, 
        "name": "frenke", 
        "primary_color": "", 
        "secondary_color": "",
        "role": 1
    }, 
    {
        "id": 7369, 
        "name": "Hurricanes", 
        "primary_color": "#ed2637", 
        "secondary_color": "#ffffff",
        "role": 2
    }
```

File: openapi.yml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    Activity:
+      type: object
+      properties:
```
Comment (joerichsen) [inline openapi.yml:156]: The properties should correspond to the following data

```
{
    "comment": "", 
    "endtime": "", 
    "id": 256592, 
    "name": "Tur til Tyskland", 
    "pickup_place": "", 
    "pickup_time": "", 
    "place": "", 
    "starttime": "2012-01-13T01:00:00+01:00", 
    "status": "tilmeldt",
    "action_method": "PUT", 
    "action_path": "/v1/activities/256592/activities_users/2363897", 
    "actions": []
}
```

File: openapi.yml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        description:
+          type: string
+        date:
+          format: date-time
+          type: string
+        location:
+          type: string
+    Member:
+      type: object
+      properties:
```
Comment (joerichsen) [inline openapi.yml]: The properties should correspond to the following data structure:

```
 { "address" : " ",
    "email" : "info@holdsport.dk",
    "id" : 78776,
    "mobile" : "12345678",
    "name" : "Demo Demo",
    "profile_picture_path": "", 
    "role" : 2
  },
  { "address" : "8000 Aarhus",
    "email" : "fran@juj.dk",
    "id" : 79413,
    "mobile" : "",
    "name" : "Kkk Kkkk"
    "profile_picture_path": "/media/BAhbB1sHOgZmSSI4MjAxMy8wg", 
    "role" : 1
  }
  ```

File: openapi.yml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+        role:
+          type: string
+        email:
+          format: email
+          type: string
+        phone:
+          type: string
+    Note:
+      type: object
+      properties:
```
Comment (joerichsen) [inline openapi.yml]: The properties should correspond to the following data structure:

```
{
        "attachment_path": null, 
        "body": "Det gik rigtig godt i l\u00f8rdags :-)", 
        "created_at": "2013-01-06T23:30:38+01:00", 
        "created_by": "Ole Kristensen", 
        "title": "Kampreferat"
    }
```

**Review by joerichsen — CHANGES_REQUESTED** [review]
This is great :-) Thanks a lot.

The components do not match the actual data structure returned by the API in this PR. I have added some comments about that.

When that is fixed I will merge this PR.

File: openapi.yml (hunk 1 above)
Comment (NewtTheWolf) [reply openapi.yml:143]: done in latest commit

File: openapi.yml (hunk 3 above)
Comment (NewtTheWolf) [reply openapi.yml]: done in latest commit

File: openapi.yml (hunk 4 above)
Comment (NewtTheWolf) [reply openapi.yml]: done in latest commit

**Comment by joerichsen** [comment]
That's great - thanks a lot - I will merge the PR now :-)

**Pull request merged**